app/src/main/java/com/summer/practice/tvtracker/Adapter.kt

```diff
@@ -0,0 +1,57 @@
+package com.summer.practice.tvtracker.popular
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.summer.practice.tvtracker.databinding.ItemMovieSimpleBinding
+import com.summer.practice.tvtracker.topRated.Movie
+
+class Adapter(
```

Comment:
It seems like this is the only adapter used for Popular & Top Rated screens, please rename it into MovieAdapter